File: ruby/testcode/benchmark_test_00500.rb
require_relative 'shared'

def handler(req)
  input = req.param('input')
  label = req.param('label', default: 'Value')
  row = sprintf("<tr><td>%s</td><td>%s</td></tr>", label, input)
  table = "<table><thead><tr><th>Label</th><th>Data</th></tr></thead><tbody>#{row}</tbody></table>"
  BenchmarkResponse.html(table)
end

File: rust/testcode/benchmark_test_00500.rs
pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let val: u64 = req.param("value").parse().unwrap_or(0);
    let signed = val as i32;
    super::shared::BenchmarkResponse::ok(&format!("signed={}", signed))
}
